python/sglang/srt/layers/attention/flashattention_backend.py

```diff
@@ -346,6 +356,8 @@ def init_forward_metadata(self, forward_batch: ForwardBatch):
                 metadata.page_table = forward_batch.req_to_token_pool.req_to_token[
                     forward_batch.req_pool_indices, : metadata.max_seq_len_k
                 ]
+
+                self._init_local_attn_metadata(metadata, device)
```

Comment:
Hey @CatherineSue for this line, have we tested this for speculative decoding topk = 1 case? Or maybe we don't have the llama eagle model yet for testing. Thanks!

Reply:
We don't have the eagle model for testing yet. Is there any concern with topk=1?